autotest/test_interface.py

```diff
@@ -323,3 +323,46 @@ def callback(sim, step):
             run_simulation(so, test_pth, callback)
         except Exception as e:
             raise Exception(e)
+
+
+def test_rhs_hcof_advanced(tmpdir):
+    def callback(sim, step):
+        model = sim.test_model
+        if step == Callbacks.timestep_start:
+
+            wel = model.wel
+            rhs = wel.rhs
+            rhs[0:3] = [-150, -100, -50]
+            wel.rhs = rhs
+
+            rhs2 = wel.get_advanced_var("rhs")
+            if not np.allclose(rhs, rhs2):
+                raise AssertionError("rhs variable not being properly set")
```

Comment:
Rather than explicitly raising `AssertionError`, use something like [numpy.testing.assert_allclose](https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_allclose.html) instead, e.g.:
```suggestion
            np.testing.assert_allclose(rhs, rhs2, err_msg="rhs variable not being properly set")
```
(`err_msg` is optional, and usually used to help guide non-obvious behavior).

Same comment recommended for L350 and L357.

Reply:
@mwtoews updated the test to use numpy testing